packages/wdio-devtools-service/src/utils.js

```diff
@@ -128,3 +129,27 @@ export function quantileAtValue (median, falloff, value) {
     const standardizedX = (Math.log(value) - location) / (Math.SQRT2 * shape)
     return (1 - internalErf_(standardizedX)) / 2
 }
+
+/**
+ * check if browser version is lower than `minVersion`
+ * @param {object} caps capabilities
+ * @param {number} minVersion minimal chrome browser version
+ */
+export function isBrowserVersionLower (caps, minVersion) {
+    const browserVersion = getChromeMajorVersion(caps[VERSION_PROPS.find(prop => caps[prop])])
```

Comment:
```suggestion
    const chromeVersion = caps[VERSION_PROPS.find(prop => caps[prop])]
    
    if (!chromeVersion) {
        log.warn('No Chrome version can be detected in the capabilities')
        return true
    }
    
    const browserVersion = getChromeMajorVersion(chromeVersion)
```

Reply:
I don't think we need to warn that no browser version is provided, it's not mandatory

Reply:
you right

Reply:
the function returns false if there is no version provided OR browser version is above 63.

otherwise it returns `true`.